plenum/server/replica.py

```diff
@@ -2188,6 +2206,10 @@ def _clear_prev_view_pre_prepares(self):
             if v < self.viewNo:
                 self.prePreparesPendingPrevPP.pop((v, p))
 
+        for (pp, sender) in self.prePreparesPendingPrevPP:
+            if pp.viewNo < self.viewNo:
+                del self.pre_prepare_tss[pp, sender]
```

Comment:
I think we need to delete it in `_gc` method together with other dicts, see `to_clean_up`. Is it sufficient to remove it only in `_gc` to avoid a lot of dels of this dict in different places?